docs/about/digital.md

```diff
@@ -102,21 +102,21 @@ To facilitate reuse and analysis of the Project's edited and encoded text of Ms.
+Static site generation thus offers a number of advantages. It allows individualized software decisions and better control and maintenance of the site. Furthermore, open-source web servers like Apache and Nginx can very efficiently serve large amounts of static data and respond to increased user traffic, leading to lower server costs. Because the technology is so fundamental to the web, techniques for serving static sites are likely to persist into the future as well as be universally understood by IT professionals tasked with keeping the assets online.
 
-The biggest challenge in developing a website using this approach is that one needs to adopt or build an appropriate static site generator for the project. There are a number of existing static site generators including [Jekyll](https://jekyllrb.com/), the most well known. Jekyll powers [Github Pages](https://pages.github.com/) which allows a user to easily turn a Github repository into a static website. While Jekyll was initially considered, the specific requirements and desired features of the Edition led to the decision to build our own system using the Ruby programming language.
+The biggest challenge in developing a website using this approach is that one needs to adopt or build an appropriate static site generator for the project. There are a number of existing static site generators including the most well known, [Jekyll](https://jekyllrb.com/). Jekyll powers [Github Pages](https://pages.github.com/) which allows a user to easily turn a Github repository into a static website. While Jekyll was initially considered, the specific requirements and desired features of the Edition led to the decision to build our own system using the Ruby programming language.
 
-The content for _Secrets of Craft and Nature_ involved the scholarly work of a number of different editors, teams, and collaborators working over several years to explore the intricacies of a complex historical text, Ms. Fr. 640. To facilitate the collaborative methods, most of this data was worked on in the Project's Google Drive. As the Project evolved and came closer to digital publication, more and more data was moved to Github repositories. At the same time, content was still under developement, and there was a need to draw from both Github and Google Drive at build time.
+The content for _Secrets of Craft and Nature_ involved the scholarly work of a number of different editors, teams, and collaborators working over several years to explore the intricacies of a complex historical text, Ms. Fr. 640. To facilitate the collaborative methods, most of this data was processed in the Project's Google Drive. As the Project evolved and came closer to digital publication, more and more data was moved to Github repositories. At the same time, content was still under development, and there was a need to draw from both Github and Google Drive at build time.
```

Comment:
Terry and I struggled with the wording here. As you mentioned, "processed" has a different meaning and should not be used here. Perhaps "edited"? Or leave as "worked on"?

Reply:
DECISION: 
"worked on"